firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -56,7 +55,7 @@
   private final Clock clock;
   private final Utils utils;
 
-  private static final long AUTH_TOKEN_EXPIRATION_BUFFER = 3600L; // 1 hour
+  private static final long AUTH_TOKEN_EXPIRATION_BUFFER_IN_SECS = 3600L; // 1 hour
 
   private CountDownLatch latch = new CountDownLatch(1);
```

Comment:
Do not make the latch global variable.

If this is only useful for getAuthToken call, then make it a local variable in there.

Reply:
Updated the code as per @ciarand comments. PTAL. Thanks.